src/test/java/uk/gov/justice/hmpps/prison/service/receiveandtransfer/ExternalMovementTransferServiceTest.kt

```diff
@@ -441,4 +455,448 @@ internal class ExternalMovementTransferServiceTest {
       }
     }
   }
+
```

Comment:
used same pattern for the other tests in the class - not very interesting and not giving a lot of value